Source/WebKit/UIProcess/mac/PageClientImplMac.mm

```diff
@@ -462,27 +462,30 @@ - (NSCursor *)_cursorRectCursor;
     
 IntPoint PageClientImpl::screenToRootView(const IntPoint& point)
 {
-    NSPoint windowCoord = [[m_view window] convertPointFromScreen:point];
-    return IntPoint([m_view convertPoint:windowCoord fromView:nil]);
+    RetainPtr view { m_view.get() };
```

Comment:
The pattern of using curly brackets like this for local variable initialization is a bit unusual. Initializing with `= ` is way more common and it'd be nice to follow our usual pattern in future patches for consistency, even though equivalent.